### ISSUE TITLE
Build failure with mingw

### ISSUE THREAD

**Issue by star-hengxing** [alembic/alembic#459]
version: 1.8.7

```console
C:\msys64\mingw64\bin\x86_64-w64-mingw32-g++.exe -DALEMBIC_DLL -DALEMBIC_EXPORTS -DPLATFORM=LINUX -DPLATFORM_LINUX -IC:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/lib -IC:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/build_9cc651d1/lib -isystem C:/Users/star/AppData/Local/.xmake/packages/i/imath/v3.1.12/34aff152d32543bc9718049db9677bc5/include -isystem C:/Users/star/AppData/Local/.xmake/packages/i/imath/v3.1.12/34aff152d32543bc9718049db9677bc5/include/Imath -m64 -fvisibility=hidden -m64 -g -std=gnu++14 -MD -MT lib/Alembic/CMakeFiles/Alembic.dir/Ogawa/OStream.cpp.obj -MF lib\Alembic\CMakeFiles\Alembic.dir\Ogawa\OStream.cpp.obj.d -o lib/Alembic/CMakeFiles/Alembic.dir/Ogawa/OStream.cpp.obj -c C:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/lib/Alembic/Ogawa/OStream.cpp
C:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/lib/Alembic/Ogawa/OStream.cpp: In constructor 'Alembic::Ogawa::v12::OStream::PrivateData::PrivateData(const std::string&)':
C:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/lib/Alembic/Ogawa/OStream.cpp:53:43: error: 'CP_UTF8' was not declared in this scope
   53 |         int wLength = MultiByteToWideChar(CP_UTF8, 0, fileName.c_str(), -1, NULL, 0);
      |                                           ^~~~~~~
C:/Users/star/AppData/Local/.xmake/cache/packages/2411/a/alembic/1.8.7/source/lib/Alembic/Ogawa/OStream.cpp:53:23: error: 'MultiByteToWideChar' was not declared in this scope
   53 |         int wLength = MultiByteToWideChar(CP_UTF8, 0, fileName.c_str(), -1, NULL, 0);
      |                       ^~~~~~~~~~~~~~~~~~~
```

`OStream.cpp` seems to be missing `#include <Windows.h>`

**Comment by lamiller0** [comment]
Yup that was it, PR for it is here: https://github.com/alembic/alembic/pull/460

**Comment by lamiller0** [comment]
This was released in the Alembic 1.8.8
